assets/.browserslistrc

```diff
@@ -0,0 +1,12 @@
+# DUP
+ChromeAndroid 61
+# Sectional (Solari)
+Firefox 90
+# Sectional (GDS)
+Firefox 126
+# Pre-Fare
+Chrome 126
+# Bus Shelter
+Firefox 126
+# Bus E-Ink
```

Comment:
Maybe this should read "Bus E-ink (GDS)" — we're replacing bus e-ink with Mercury displays eventually, and then this would not be relevant to them.